src/views/ResultsPage/PrintModalContent.jsx

```diff
@@ -1,20 +1,17 @@
 import React, { useEffect } from 'react';
 import PropTypes from 'prop-types';
-import { useSelector } from 'react-redux';
 import { usePrintApi } from '../../hooks';
 import { buildQueryString } from '../../utilities';
 import { useAppSettings } from '../../store/store.js';
 
 const queryString = require('query-string');
 
-const PrintModalContent = ({ selectedList = [] }) => {
+const PrintModalContent = ({ selectedList = [], searchCriteriaObject }) => {
```

Comment:
NIT: just realized, selected list is required, no need for defaulting to empty string.

Reply:
done!